components/button/src/button/button.js

```diff
@@ -24,6 +25,7 @@ export const Button = ({
     onBlur,
     onClick,
     onFocus,
+    isLoading,
```

Comment:
While I like having booleans/predicates start with `is` and `has`, existing components use `loading` so that's probably what we should use here (e.g. https://github.com/dhis2/ui/blob/master/components/select/src/single-select/single-select.js#L162).

Reply:
Thanks for pointing that out! Agreed that consistency is good. Fixed.